File: Ej 4 (Es Primo)/Java/EsPrimo.java
public class EsPrimo {
    public static boolean esPrimo(int num){
        if(num == 2 || num == 7 || num == 5 || num == 3 || num == 11){
            return true;
        }else if(num % 2 == 0||num % 3 == 0||num % 5 == 0||num % 7 == 0||num % 11 == 0){
            return false;
        }else{
            return true;
        }
    }
    public static void main(String[] args) {
        for(int i = 1; i < 101; i++){
            if(esPrimo(i)){
                System.out.println(i);
            }
        }
    }
}

File: Ej 4 (Es Primo)/Python/EsPrimo.py
## DeathbatO

def Es_Primo(num):
    if num == 2 or num == 3 or num == 5 or num == 7 or num == 11:
        return True
    elif num % 2 == 0 or num % 3 == 0 or num % 5 == 0 or num % 7 == 0 or num % 11 == 0:
        return False
    else:
        return True
    
for i in range(100):
    if Es_Primo(i):
        print(i)